### DISCUSSION TITLE
Works perfectly with Buds Live!

### DISCUSSION THREAD

**Discussion by ktheod** [timschneeb/GalaxyBudsClient#167] [General]
I know it is not a useful discussion, but just giving a Star wasn't enough!

I actually decided to get the Buds Live because this project reassured me that I will be able to use them with Windows devices.

Apart from the app that works perfectly with Buds Live, it is also very useful to force-connect the buds if they are already connected to another device!

Thanks for creating this! 

ps. if you want to delete this after reading to keep discussions clear, feel free. 

:)

**Comment by Untitled-code** [comment]
Yes, very good app for Live. It works perfectly with LInux. Is there any chance to have this app on Android? Thanks
